ds-card-reader/res/configuration.toml

```
@@ -49,12 +49,12 @@ Type = "consul"
   Host = "localhost"
   Port = 59881
 
-[Driver]
-  DeviceName = 'card-reader'
-  DeviceSearchPath = '/dev/input/event*'
-  VID = '65535' # 0xFFFF
-  PID = '53'    # 0x0035
-  SimulateDevice = 'true'
+[DriverConfig]
+  DeviceName = "card-reader"
```

```suggestion
  DeviceName = "ds-card-reader"
```
ds-card-reader is the name expected by the other services.

---

The /res/devices/device-list.toml "Name = 'ds-card-reader' also needs to be updated